src/pages/Registration/Steps/Documentation/FSA/Form/index.tsx

```diff
@@ -71,7 +71,7 @@ export default function Form(props: Props) {
 
       <div className="grid grid-cols-2 sm:flex gap-2 mt-8">
         <Link
-          aria-disabled={isSubmitting}
+          aria-disabled={isSubmitting || isRedirecting}
```

Comment:
Noticed the button becoming "enabled" when the form was redirecting, so added this check to keep it disabled.